### ISSUE TITLE
Contents under a sub directory of a symlinked directory doesn't show up in CocosCreator on Mac

### ISSUE THREAD

**Issue by sucineri** [cocos/cocos-engine#2054]
Repo steps:
1. Create a  new project in CocosCreator. 
2. Create a symlink to a directory that contains at least 1 sub directories, or run the following from the project's directory to create some test data:
```
mkdir -p  ~/Desktop/sl_test_layer_1/sl_test_layer_2/
touch ~/Desktop/sl_test_layer_1/sl_test_layer_2/sl_test.json
ln -s ~/Desktop/sl_test_layer_1/ ./assets/symlink
```

3. Go back to CocosCreator to observe the asset appears.
![screen shot 2017-10-18 at 1 39 33 pm](https://user-images.githubusercontent.com/6344189/31741689-ff426124-b409-11e7-924c-7210d7a8cfe7.png)

4. Close and open the project again.
5. Observe that the asset under the sub directory doesn't appear anymore.
![screen shot 2017-10-18 at 1 40 27 pm](https://user-images.githubusercontent.com/6344189/31741713-1174fb22-b40a-11e7-9574-f2694077d1fc.png)

**Comment by jareguo** [comment]
It should be a bug of node.js... maybe we can take a look

**Comment by xianyinchen** [comment]
This problem has been solved in version 2.4
